govc/fields/ls.go

```diff
@@ -39,13 +40,16 @@ func init() {
 func (cmd *ls) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
 	cmd.ClientFlag.Register(ctx, f)
+
+	f.StringVar(&cmd.mot, "t", "", "Filter by Object Type")
```

Comment:
cosmetic: in other commands we name the var `kind`, flag `-type` and description "Managed object type", for example:

https://github.com/vmware/govmomi/blob/c0f4e9754e39f3910357a6e3e56bbfb387b35a65/govc/fields/set.go#L43

https://github.com/vmware/govmomi/blob/c0f4e9754e39f3910357a6e3e56bbfb387b35a65/govc/fields/add.go#L42

Though I know elsewhere in the govc commands we have a mix of `-t` and `-type`

Reply:
done.